### ISSUE TITLE
Storage class NFS-rwx

### ISSUE THREAD

**Issue by aminshah1** [grantjenks/python-diskcache#249]
Hi, I see this line fails in multi-threading scenarios:
`executing file "/opt/venv/lib/python3.8/site-packages/diskcache/core.py", line 666, in _execute_with_retry`
We're using a persistent volume with NFS-rwx storage class. I'm wondering if this error is related to storage class we use? If I can switch to another storage class, what's your recommendation? Thanks.

**Comment by grantjenks** [comment]
I’m not much help here. No idea what NFS-rwx is? If it’s related to Network File System (NFS) then it probably won’t work there due to sqlite issues. See the caveats: https://grantjenks.com/docs/diskcache/tutorial.html#caveats

**Comment by aminshah1** [comment]
Yes, it's Network File System (NFS) with ReadWriteMany access. Thanks for the link. Looks like NFS is the source of issue.
